mmv1/provider/terraform_kcc.rb

```diff
@@ -87,76 +87,100 @@ def copy_common_files(output_folder, generate_code, generate_docs) end
     def guess_metadata_mapping_name(object)
       # Split the last import format by '/' and take the last part. Then use
       # the regex to verify if it is a value field in the format of {{value}}.
-      last_import_part = import_id_formats_from_resource(object)[-1].split('/')[-1].scan(/{{[[:word:]]+}}/)
+      last_import_part =
+        import_id_formats_from_resource(object)[-1].split('/')[-1].scan(/{{[[:word:]]+}}/)
       # If it is a value field, the length of last_import_part will be 1;
       # otherwise it'll be 0.
       # Remove '{{' and '}}' and only return the field name.
       last_import_part.first.gsub('{{', '').gsub('}}', '') if last_import_part.length == 1
     end
 
-    def is_server_generated_name(name, object)
-      has_computed_name_configured = object.custom_code.post_create == 'templates/terraform/post_create/set_computed_name.erb' if object.custom_code.post_create
+    def server_generated_name?(name, object)
```

Comment:
This name change was manual. This is called from the template, so that was changed as well.

Reply:
Looks good, thx!